logbook-servlet/src/main/java/org/zalando/logbook/servlet/LocalResponse.java

```diff
@@ -1,22 +1,26 @@
 package org.zalando.logbook.servlet;
 
-import org.zalando.logbook.HttpResponse;
-import org.zalando.logbook.Origin;
-import org.zalando.logbook.RawHttpResponse;
-
-import javax.servlet.ServletOutputStream;
-import javax.servlet.http.HttpServletResponse;
-import javax.servlet.http.HttpServletResponseWrapper;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
 import java.io.OutputStreamWriter;
 import java.io.PrintWriter;
 import java.nio.charset.Charset;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
```

Comment:
Is it used?

Reply:
SimpleDateFormat?

Reply:
I organized imports again. Should be gone now.

Reply:
Yes, for me it is a red flag because it is not thread safe and java 8 has better alternative